styles/gmail/catppuccin.user.css

```diff
@@ -454,6 +454,45 @@
         background: lighten(@accent-color, 10%) !important;
       }
     }
+
+    /* loading */
+    #explosion_clipper_div > .la-i > div {
+      background: @surface0 !important;
+    }
+
+    #nlpt {
+        background-color: @surface0 !important;
+    }
+
+    #nlpt::before {
+        background-color: @overlay0 !important;
+    }
+
+    .la-b > .la-l, .la-b > .la-r, .la-b > .la-m {
+        background: @surface1 !important;
+    }
+
+    .la-k .la-l, .la-k .la-r {
+        border-color: @base !important;
+    }
+
+    .la-k .la-m {
+        background: @base !important;
+        clip-path: polygon(47% 100%, 100% 47%, 100% 100%);
```

Comment:
```suggestion
```

Reply:
No, no. That is very much needed to look correctly. I've added it myself.

Reply:
It shouldn't be needed AFAIK, strange...

Reply:
Here is the difference.

https://github.com/catppuccin/userstyles/assets/27358071/5d5098cb-40b4-4703-accd-b47c9666eeab